src/components/fields/edit/SpecifiedTimeEditor.jsx

```diff
@@ -18,14 +18,7 @@ import Text from '../../Text';
 import FormCore from './FormCore';
 
 export default function SpecifiedTimeEditor(props) {
-  const {
-    schema,
-    value,
-    onChange,
-    width = 280,
-    minimalLabels = false,
-    ...rest
-  } = props;
+  const { schema, value, onChange, width = 280, ...rest } = props;
```

Comment:
Okay, I think I remember now. If you remove `minimalLabels` it becomes part of `rest` and gets stuck on the DOM element. So deconstructing the variable prevents a console warning